legion_airflow/legion_airflow/hooks/s3_hook.py

```diff
@@ -194,6 +212,98 @@ def copy_folder(self, src_bucket: str, src_key: str, dest_bucket: str, dest_key:
                                  .format(self.bucket_prefix + src_bucket, key_from, dest_bucket, key_to))
                 dist_obj.copy(source)
 
+    def load_file(self,
+                  filename,
+                  key,
+                  bucket_name=None,
+                  replace=False,
+                  encrypt=False):
+        """
+        Load a local file to S3
+
+        :param filename: name of the file to load.
+        :type filename: str
+        :param key: S3 key that will point to the file
+        :type key: str
+        :param bucket_name: Name of the bucket in which to store the file
+        :type bucket_name: str
+        :param replace: A flag to decide whether or not to overwrite the key
+            if it already exists. If replace is False and the key exists, an
+            error will be raised.
+        :type replace: bool
+        :param encrypt: If True, the file will be encrypted on the server-side
+            by S3 and will be stored in an encrypted form while at rest in S3.
+        :type encrypt: bool
+        """
+        if not bucket_name and not self.bucket_prefix:
+            (bucket_name, key) = self.parse_s3_url(key)
+
+        with self.open_file(bucket_name, key, 'w') as dist:
+            with open(filename, 'r') as source:
+                for line in source:
+                    dist.write(line)
+
+    def load_string(self,
```

Comment:
load here is a bit confusing. Would it be better to name it store() or upload() ?